DataFormats/Common/interface/RefItemGet.h

```diff
@@ -115,6 +115,26 @@ namespace edm {
   inline bool isThinnedAvailable(RefCore const& product, KEY const& iKey) {
     return refitem::IsThinnedAvailableImpl<C, KEY>::isThinnedAvailable_(product, iKey);
   }
+
+  /// Return a Ref to thinned collection corresponding to an element of the Ref to parent collection
+  //
+  // The thinned may point to parent collection, in which case the Ref-to-parent is returned
+  //
+  // If there are no thinned collections containing the element of the Ref-to-parent, a Null Ref is returned.
```

Comment:
Given that the "thinned collection" is fully defined, this should be something along
```suggestion
  // If thinned does not contain the element of the Ref-to-parent, a Null Ref is returned.
```

Reply:
Done